README.md

```diff
@@ -193,8 +193,8 @@ For local development:
 1. Make the decision whether you want to run this using your local environment, or develop from within a docker image.
 Developing within a docker image is recommended, since that most closely models how users will use this. Additionally, there are some issues with installing the requirements.txt on mac.
 If you don't wish to run this within a docker image, skip to step 5.
-2. run `docker pull us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.12`
-3. run the image from *one directory above* the root directory of this repo via `docker run -itd --entrypoint='/bin/bash' -v $PWD/terra-notebook-utils:/work -u root -e PIP_USER=false --name test-image us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.12`
+2. run `docker pull us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:1.1.5`
```

Comment:
Could you add a comment explaining that we should use the image currently on Terra? Right now it is 1.1.5 but his can change at any point. We should use the version shown by Terra UI

Reply:
Good point! Is there a link I can add here that shows the version that Terra UI currently uses?

Reply:
The source of truth is always going to be this json file, which the UI reads from https://console.cloud.google.com/storage/browser/_details/terra-docker-image-documentation/terra-docker-versions-candidate.json;tab=live_object?project=broad-dsp-gcr-public